spec/spec_helper.rb

```diff
@@ -48,7 +48,7 @@ module BlockScore
     Kernel.srand config.seed
 
     config.around do |example|
-      Timeout.timeout(10, &example)
+      Timeout.timeout(6, &example)
```

Comment:
The lower this timeout is the better from a mutation testing perspective, as infinite loops are killed earlier.

It comes as a downside that a loop might be killed for "assumed infinite loop" where it would actually terminate with a green test so an alive mutation. So there is a trade off.